src/conn/conn_layered.c

```diff
@@ -368,7 +368,9 @@ __disagg_pick_up_checkpoint(WT_SESSION_IMPL *session, uint64_t meta_lsn)
      */
 
     /* Read the checkpoint metadata of the shared metadata table from the special metadata page. */
-    WT_ERR(__disagg_get_meta(session, WT_DISAGG_METADATA_MAIN_PAGE_ID, meta_lsn, &item));
+    WT_ERR_MSG_CHK(session,
+      __disagg_get_meta(session, WT_DISAGG_METADATA_MAIN_PAGE_ID, meta_lsn, &item),
+      "Disagg metadata fetching failed, with lsn : %" PRIu64 "", meta_lsn);
```

Comment:
Do you need these extra quotes?
```suggestion
      "Disagg metadata fetching failed, with lsn : %" PRIu64, meta_lsn);
```